doc/gemma_online.md

```diff
@@ -0,0 +1,34 @@
+
+Deze nieuwe uitwisselingsstandaarden zijn niet alleen technisch anders van opzet, ze worden ook volgens het Common Ground principe door gemeenten zelf stapsgewijs opgezet en ingevuld naar behoefte. Op basis van zgn. user stories wordt beschreven welke functionele wensen gemeenten hebben die door een standaard ondersteund moet worden.
+
+Een user story is een korte beschrijving (story) van wat een gebruiker (user) wil. User stories worden gebruikt bij het ontwikkelen van software of producten. Een user story bestaat uit enkele zinnen gewone spreektaal van de (computer)gebruiker waarin staat wat de gebruiker doet of moet doen, als onderdeel van z'n werk. (bron: https://nl.wikipedia.org/wiki/User_story) 
+
+Door in korte, heldere taal een wens te beschrijven en deze in te vullen sluit de standaard naadloos aan op de behoeften van gemeenten zonder dat er overbodige functionaliteit en techniek opgenomen wordt. Voor leveranciers is het daarom ook helder waar de standaard voor dient en hoe deze toegepast dient te worden.
+
+Naast gebruik van user stories worden standaarden ook agile ontwikkeld. In plaats van een langdurig ontwikkelproces wordt een onderdeel van een standaard in een korte cyclus uitgewerkt en afgestemd met de gebruikers. 
+
+== Transparantie
+De API standaarden worden volledig transparant en openbaar ontwikkeld. Alle user stories, documentatie, broncode, ontwerpen en andere zaken worden in een openbare Github repository vastgelegd. Daar kunnen belanghebbenden ook zelf userstories en bevindingen doorgeven die door het ontwikkelteam besproken, beoordeeld en opgepakt kunnen worden. 
+
+
+== Zaakgericht Werken
+De eerste standaard die op deze manier wordt ontwikkeld is een groep standaarden ter ondersteuning van het <a href="https://www.gemmaonline.nl/index.php/Thema_Zaakgericht_werken">Zaakgericht Werken</a>. De opvolger van de <a href="https://www.gemmaonline.nl/index.php/Zaak-_en_Documentservices">Zaak- Documentservices</a>, een op <a href="https://www.gemmaonline.nl/index.php/Sectormodel_Zaken:_StUF-ZKN">StUF ZKN</a> gebaseerde koppelvlakstandaard, is opgezet volgens de <a href" https://www.gemmaonline.nl/index.php/GEMMA_Architectuur">GEMMA 2</a> referentie architectuur. 
+
+De Github repository waar deze standaard in ontwikkeld wordt staat hier: <a href="https://github.com/vng-realisatie/gemma-zaken">https://github.com/vng-realisatie/gemma-zaken</a>. De Github repository is dus een ontwikkelomgeving, in software termen zou dit zelfs een nightly build genoemd kunnen worden. De versie van de standaard in ontwikkeling zoals die getoond wordt tijdens een sprint demo staat op een aparte omgeving welke te vinden is op <a href="https://ref.tst.vng.cloud/">https://ref.tst.vng.cloud/</a> en is te vergelijken met een stable build.
```

Comment:
Ik zou het gedeelte van "nightly" weglaten, niet echt correct en alleen maar verwarrend.

Ter info, staan releases straks netjes onder elkaar: https://github.com/maykinmedia/demo-api/releases
De laatste "development" versie is altijd te vinden op de `master` branch, te vinden op: https://github.com/maykinmedia/demo-api/